Ajout html creation de triplet

maximelodato opened the pull request.

maximelodato committed:
creation d'une vue pour cree des triplets

jeremie-olivier committed:
fix graph API controller Cross origin

maximelodato committed:
vue charger tous les Atoms (de type Triples ou Atoms)
 Lorsque nous tapons dans un champ, le résultat est filtré pour n’afficher que des résultats similaires
 Un bouton ** Créer un triplet ** enregistre les nouveaux triplets créés dans la base de données
 Les triplets nouvellement créés apparaîssent dans la liste des atomes disponibles

maximelodato committed:
fix graph.api.v1 et ajout de la fonctionnalité de recherche et d'affichage des triplets

maximelodato committed:
remise en fonction de l'affichage de l'api graph v1

maximelodato committed:
Si un triplet existe déjà, afficher un message d'erreur et supprimer les underscores dans les atomes du tableau HTML.

Il y avait une erreur lors de la création de certains triplets ; j'ai dû modifier certaines lignes de l'API.

maximelodato committed:
traduction des atoms dans les seeds en anglais

AlexandreTedesco committed:
rename files

jeremie-olivier commented on app/models/triple.rb:
```diff
@@ -15,10 +15,11 @@ class Triple < ApplicationRecord
 
   # Computed ID based on components
   before_validation :set_computed_id
+  validates :id, uniqueness: { message: "Ce triplet existe déjà" }
```
This should be in english

AlexandreTedesco committed:
added possibility to choose a seed to populate database

maximelodato committed:
modif francais > anglais triple.rb L18 (This triplet already exists)

maximelodato committed:
Merge branch 'ajout-html-creation-de-triplet' of github.com:THP-Lab/rails-mock-api into ajout-html-creation-de-triplet

maximelodato committed:
Generation of 2 CSV files: (1) Atom and (2) Triplet for the data populator.

maximelodato committed:
changement de certain mots de francais a anglais dans la partie vue trple index.html.erb

maximelodato committed:
changement de certain mots de francais a anglais dans la partie vue trple index.html.erb

maximelodato committed:
Utiliser un triple complet comme valeur dans n'importe quel champ (sujet, prédicat ou objet)
Voir clairement les triples complets disponibles dans une section dédiée des suggestions
Cliquer sur n'importe quelle cellule du tableau pour utiliser le triple complet correspondant

maximelodato committed:
Utiliser un triple complet comme valeur dans n'importe quel champ (sujet, prédicat ou objet)
Voir clairement les triples complets disponibles dans une section dédiée des suggestions
Cliquer sur n'importe quelle cellule du tableau pour utiliser le triple complet correspondant

maximelodato committed:
text en anglais dans index.html.erb

maximelodato committed:
text en anglais dans index.html.erb

maximelodato committed:
modif des seed pour etre en accord avec les csv pour l'utilisation sur le intuition Data Populator

maximelodato committed:
Création du modèle Person pour gérer les entités de type personne
Création de la table 'persons' avec les champs nécessaires
Amélioration du modèle Triple pour gérer correctement les références aux atomes et aux entités typées
Le seed CSVPopulator fonctionne maintenant correctement, important les données depuis atoms.csv et triples.csv dans la base de données.

maximelodato committed:
Merge branch 'ajout-html-creation-de-triplet' of github.com:THP-Lab/rails-mock-api into ajout-html-creation-de-triplet

maximelodato commented:
This code has already been merged into another PR after being reviewed
https://github.com/THP-Lab/rails-mock-api/pull/15
GitHub
